src/test/utils/init-api-client.js

```diff
@@ -38,6 +38,9 @@ const onPreFormatRequestOptions = (request) => {
         }
     }
 
+    //Add fake remote IP address
+    request.remoteAddress = 'mochatest';
```

Comment:
I know this was my code, but we should update the remoteAddress to be 'mochatestIP'